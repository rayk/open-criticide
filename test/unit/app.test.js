/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../src/gui/app';

describe('Output HTML for the application.', () => {
  test('Welcome message snapshot.', () => {
    const component = renderer.create(<App />);
    const htmlTree = component.toJSON();
    expect(htmlTree).toMatchSnapshot();
  });
});
