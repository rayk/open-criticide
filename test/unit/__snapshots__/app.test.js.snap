// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output HTML for the application. Welcome message snapshot. 1`] = `
<div>
  <h2>
    Welcome to Criticide App.
  </h2>
</div>
`;
